src/lib/queries/ibc.ts

```diff
@@ -79,21 +102,28 @@ export interface IbcExtension {
       readonly unreceivedAcks: (
         portId: string,
         channelId: string,
-        packetCommitmentSequences: readonly number[]
+        packetAckSequences: readonly number[]
       ) => Promise<QueryUnreceivedAcksResponse>;
       readonly nextSequenceReceive: (
         portId: string,
         channelId: string
       ) => Promise<QueryNextSequenceReceiveResponse>;
     };
     readonly client: {
-      readonly states: () => Promise<QueryClientStatesResponse>;
       readonly state: (clientId: string) => Promise<QueryClientStateResponse>;
-      readonly stateTm: (clientId: string) => Promise<TendermintClientState>;
+      readonly states: (
+        paginationKey?: Uint8Array
+      ) => Promise<QueryClientStatesResponse>;
       readonly consensusState: (
         clientId: string,
         height?: number
       ) => Promise<QueryConsensusStateResponse>;
+      readonly consensusStates: (
+        clientId: string,
+        paginationKey?: Uint8Array
+      ) => Promise<QueryConsensusStatesResponse>;
+      readonly params: () => Promise<QueryClientParamsResponse>;
+      readonly stateTm: (clientId: string) => Promise<TendermintClientState>;
```

Comment:
Ah, there it is, just reordered. It would be nice to have statesTm() in plural,but please feel free to add that with the auto-pagination